components/deck/WordCardEdit.js

```diff
@@ -17,6 +17,13 @@ const WordCard = styled.div`
   padding: 20px;
   margin-top: 20px;
   position: relative;
+
+  :hover {
+    /* background-color: ${({ theme }) => theme.colors.focusLight}; */
+  }
```

Comment:
Is this something that we want to have later? Otherwise it might make sense to remove this.